tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -232,6 +305,7 @@ def test_dhcp_relay_default(ptfhost, dut_dhcp_relay_data, validate_dut_routes_ex
                 loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix="dhcpmon counter")
                 marker = loganalyzer.init()
                 loganalyzer.expect_regex = [expected_agg_counter_message]
+                init_and_verify_dhcp_relay_counters(duthost)
```

Comment:
In t0 topo, we only have one tor, then it's okay to init and verify `duthost`. But in dualtor, we have 2 tors (1 active and 1 standby), we need to verify both tor if testing_mode == DUAL_TOR_MODE
You can refer to above syslog verification for dhcpmon